app/src/main/java/fr/groggy/racecontrol/tv/f1tv/F1TvClient.kt

```diff
@@ -252,7 +252,7 @@ class F1TvClient @Inject constructor(
 
     private fun getCurrentLocale(): String {
         return when (val isO3Language = Locale.getDefault().isO3Language) {
-            "deu", "fra", "nld", "spa", "por" -> isO3Language.toUpperCase(Locale.ROOT)
+            "deu", "fra", "nld", "spa", "por", "rus" -> isO3Language.toUpperCase(Locale.ROOT)
```

Comment:
Hi tks.

However, I don't think F1 apis support Russian... Did you check that?

Otherwise this won't work

Reply:
Hi.

Got it. I hadn't noted this dependency.
Unfortunately it doesn't support.
Ok. Let's just leave messages file for the future.